wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,164 @@
+
+You would be given an option where to save your edits.
+
+- **"Commit Directly to the ``xyz`` branch"** 
+  - This will commit the changes through your branch directly.
+
+- **"Create a New Branch"** 
+  - Automatically creates a Branch for your edits. This is useful if you're making bulk edits.
+
+Pick an option and click "Commit Changes".
+
+### Through your Local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork to your PC and make changes there.
+
+if you have Git installed, enter this in your Command Line Interface (CLI):
+```prolog
+git clone https://github.com/<your github username (ie. peppy)>/osu-wiki 
+```
+*Note: If you don't have Git installed, you can go to [their website](https://git-scm.com/) to download Git.*
+
+and this should clone your fork into the folder you specified. You can now proceed to make your changes.
+
+After making your changes locally, you will need to commit this on your local copy.
+
+This can be done by entering this to the Command Line Interface (CLI): 
+```prolog
+git add .
+git commit -m "<explain your changes here>"
+```
+However, some editors (ie. Visual Studio Code) may provide a Git interface.
+
+After committing your changes, you need to push the commits to GitHub so everyone can see it. To do this, enter the following to your Command Line Interface:
+
+```prolog
+git push origin <branch>
+```
+This should prompt you to enter your GitHub login credentials. After you enter them, this should push your commits to GitHub.
+
+_Note: However, If you're authentication details were saved in Git already, no prompt will appear and will proceed directly to push._
+
+## Pull Requests
+
+After pushing your commits, you must make a "Pull Request". This lets us know that your commits are ready to be reviewed.
+
+
+Go to the "Pull Requests" tab.
+
+![Pull Request Tab](6.JPG "Pull Request Tab")
+
+This would lead you to the pull requests page.
+
+![PR UI](11.jpg "Pull Requests Page")
+
+This is the Pull Request Page. Now click on "New pull request"
+
+![Compare Changes Page](12.jpg "Comparing Changes")
+
+We want to compare ``ppy:master`` against your branch.
+
+From here, select the branch that contains your commits besides your "head fork".
+
+![Comparing Changes 2](13.jpg "Comparing Changes 2")
+
+_However, if you made changes in your master branch, the changes will be shown without the need to change anything in the head fork options._
+
+Now after viewing your changes, click "Create Pull Request" and fill it with the following details:
+
+- **Title** : the summary of the changes you have made
+  - Note: it is really important to have a meaningful and clear title, otherwise it will cause confusion
+
+- **Leave a Comment** : the details of the changes you have made
+
+![PR Creation Dialogue](14.jpg "PR Creation Dialogue")
+
+When you are finished, click on the "Create pull request" button and wait for a reviewer to check and, possibly, merge your changes.
```

Comment:
~~FINISH HIM~~

Nitpicking time: `Click on the "Create pull request" button and wait for a reviewer to check and, possibly, merge your changes to the official branch.`